resources/diff/rules/defaultRules.json

```diff
@@ -96,7 +96,7 @@
     }
   },
   {
-    "name": "Detect addition of a required parameter",
+    "name": "Parameter Added",
```

Comment:
This is addition of the required parameter. Name should be descriptive enough and indicate what the rule does exactly. This is the reason I prefer the existing names

Reply:
Thanks for pointing out that I missed a bit of important information!

I think that the old names were too long and seem more like descriptions - a name should work in multiple contexts (and ideally be short). For example, compare these two sentences:
> 3 breaking changes found: Parameter Removed, Required Parameter Added, Version Changed.

> 3 breaking changes found: Detect removal of a parameter, Detect addition of a required parameter, Detect change in API version.

I think that the first one is a lot cleaner and easier to understand.